src/fhirConfig.ts

```diff
@@ -113,9 +113,19 @@ export interface Profile {
     resources?: Resources;
 }
 
-export interface FhirConfig {
+export interface ProductInfo {
     orgName: string;
+    productVersion?: string;
+    productName?: string;
+    productSlug?: string;
+    productDescription?: string;
+    productPurpose?: string;
+    copyright?: string;
```

Comment:
Hmm I didn't know FHIR Servers can have copyright? or is it the copyright of the capability statement?

Reply:
It refers to the copyright of the Capability Statement itself: 

https://www.hl7.org/fhir/capabilitystatement-definitions.html#CapabilityStatement.copyright

Reply:
thanks for the link makes sense to me